mep/common/admin.py

```diff
@@ -80,6 +80,26 @@ def before_import_row(self, row, **kwargs):
         """
         pass
 
+    def validate_row_by_slug(self, row):
+        if not row.get("slug"):
```

Comment:
let's add a doc string to document what this is; here's a draft version with my understanding of what's going on here (may need more detail)

```suggestion
     """Make sure the record to update can be found by slug or past slug; if the slug is a past slug, row data is updated to use the current slug."""
        if not row.get("slug"):
```